src/transforms/extensions.ts

```diff
@@ -50,15 +48,8 @@ function normalizeMultipleContentTypes(codeModel: CodeModel) {
 
         for (const parameter of parameters) {
           const parameterMetadata = getLanguageMetadata(parameter.language);
-          if (
-            !parameter.protocol.http &&
-            !parameterMetadata.serializedName &&
-            parameterMetadata.name.toLowerCase() === "contenttype"
-          ) {
-            parameterMetadata.serializedName = "Content-Type";
-            const http = new Protocol();
-            http.in = ParameterLocation.Header;
-            parameter.protocol.http = http;
+          if (parameterMetadata.name.toLowerCase() === "contenttype") {
+            parameter.required = true;
```

Comment:
Nice workaround for the modelerfour issue, I'm planning to fix this so that it's no longer necessary 😅

Reply:
You'll only make it required if there are multiple media types though right? :)

Reply:
Actually, I think the issue I'm thinking of is not related to Content-Type parameters, but body parameters when there are multiple requests.  Those body parameters are not carrying forward `required` when they should be.

Reply:
Ah right, Brian mentioned that. That will make the overload signatures look much better because the optional parameter will be the same type in all overloads (the body is currently the only thing in the optional type which is why we need multiple versions of it today).